### DISCUSSION TITLE
two playlists (different urls) that happen to have a channel that have the same url then that channel only gets created once.

### DISCUSSION THREAD

**Discussion by aniel300** [euzu/tuliprox#121] [General]
it seems if two playlists (different urls) happen to have a channel that have the same url then that channel only gets created once. is this working as designed?

**Comment by aniel300** [comment]
can u make where it will create the channel again even though they have the same url but exist on two different playlist?

**Reply by euzu** [reply]
What do you mean with playlist ? 
If you have multiple inputs, and create from there  2 outputs m3u and xtream, you want in m3u another entry than in xtream ? 

I really dont get the use-case if it is the same url ? 
Or asking you to put the same entry inside different groups ?

**Reply by aniel300** [reply]
let's say i have two playlists `a` and `b` and they both have a common/same everything/url, name, etc channel on them which is `c`. right now, it will write channel `c (b)` to the end result m3u but as one entry. ideally, I would like to write it twice, especially since am using suffix which will make it where `c` exist as `c (a)` and `c (b)` in the end result playlist.

**Reply by euzu** [reply]
Then please explain to me the sense of writing exact the same channel  with different names ? I dont get it.
If you want it in different groups than this will make sense.

**Reply by aniel300** [reply]
i don't have an exact answer to this but rather am basing my request base on consistency.

**Reply by aniel300** [reply]
wait...am confused did u made any changes to this? right now is working

**Reply by euzu** [reply]
no i did not, ok if it sworking then its good:)

**Reply by aniel300** [reply]
ahaha ok

**Reply by aniel300** [reply]
> no i did not, ok if it working then its good:)


ok issue happened again. and am also getting this error int he logs, is it normal? 
```
[2024-11-21T07:07:15Z ERROR m3u_filter::processing::playlist_processor] Failed to process categories missing field `category_id`
```

**Reply by euzu** [reply]
Did this happen with xtream source/input ?

**Reply by aniel300** [reply]
the beginning of this post was for the first issue/ request and it was all m3us playlists. this error was introduced after adding xc imputs. so right now am using a combination of m3s and xc imputs.

**Reply by euzu** [reply]
i think your xtream source is not valid, because category_id should be mandatory for categories.

**Reply by aniel300** [reply]
there is few but i will take a look, thanks.
